lib/rubocop/cop/rails/redundant_active_record_all_method.rb

```diff
@@ -174,7 +174,7 @@ def possible_enumerable_block_method?(node)
           parent = node.parent
           return false unless POSSIBLE_ENUMERABLE_BLOCK_METHODS.include?(parent.method_name)
 
-          parent.parent&.block_type? || parent.parent&.numblock_type? || parent.first_argument&.block_pass_type?
+          parent.block_node || parent.first_argument&.block_pass_type?
```

Comment:
`block_node` doesn't return a boolean value for predicate method. `block_literal?` seems more appropriate.

```suggestion
          parent.block_literal? || parent.first_argument&.block_pass_type?
```

Reply:
Thank you for your review! I've fixed it.